db/migrate/20160506185102_create_games.rb

```diff
@@ -0,0 +1,12 @@
+class CreateGames < ActiveRecord::Migration
+  def change
+    create_table :games do |t|
+
+      t.integer :game_id
+      t.integer :jedi_user
```

Comment:
We're going to set it up such that:

```
g = Game.first
g.jedi_user
```

Should be the `User` class.  Making this field have the same name as what we will map it to will cause problems.  I suggest rolling back:

```
rake db:rollback
```

Then changing it to:

```
t.integer :jedi_user_id
t.integer :sith_user_id
```